test/Microsoft.ServiceFabric.Actors.Tests/Runtime/ActorManagerTests.cs

```diff
@@ -0,0 +1,345 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Fuzzy;
+using Inspector;
+using Microsoft.ServiceFabric.Actors.Diagnostics;
+using Microsoft.ServiceFabric.Actors.Tests;
+using Microsoft.ServiceFabric.Diagnostics;
+using Moq;
+using Xunit;
+
+namespace Microsoft.ServiceFabric.Actors.Runtime
+{
+    public class ActorManagerTests
```

Comment:
Test class names should be singular - ActorManagerTest, otherwise the test names don't make sense in English.